packages/sdk-react/src/components/auth/Facebook.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+
+import { useState } from "react";
+import styles from "./Socials.module.css";
+import { exchangeCodeForToken, generateChallengePair } from "./facebookUtils";
+import { sha256 } from "@noble/hashes/sha256";
+import { bytesToHex } from "@noble/hashes/utils";
+import facebookIcon from "assets/facebook.svg";
+interface FacebookAuthButtonProps {
```

Comment:
what was the resolution for this btw? it's fine if you leave it unchanged, but would be good to have some rationale posted here for posterity

Reply:
Where did my comment go... Yeah I figured that we will likely have plenty of OIDC options in the future and since they differ it would make sense to separate them out. However, I did pull the oidc fetching logic into a separate oidc.ts file. If you're talking about the props then the same idea - will likely have more Oidc options, would make sense if the props definition just lived in the same file